tests/AutoReview/TransformerTest.php

```diff
@@ -63,4 +64,40 @@ public function provideTransformerCases()
 
         return $transformersArray;
     }
+
+    /**
+     * @requires PHP 7.1
+     *
+     * @dataProvider provideTransformerPriorityCases()
```

Comment:
I don't believe `dataProvider` annotation was intended to be used with parenthesis
```php
public const REGEX_DATA_PROVIDER = '/@dataProvider\s+([a-zA-Z0-9._:-\\\\x7f-\xff]+)/';
```

and https://phpunit.de/manual/6.5/en/writing-tests-for-phpunit.html#writing-tests-for-phpunit.data-providers